clang/lib/DPCT/SyncThreadsSideEffectsAnalyzer.cpp

```diff
@@ -0,0 +1,169 @@
+//===---- SyncThreadsSideEffectAnalyzer.cpp ---------------------*- C++ -*-===//
```

Comment:
GroupFunctionAnalyzer.cpp 

add  a "//file purpose:  " section to desc the functionality.

Reply:
fixed